client/packages/coldchain/src/Equipment/DetailView/DetailView.tsx

```diff
@@ -100,12 +100,20 @@ export const EquipmentDetailView = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [data, setDraft]);
 
-  const locations =
+  let locations =
     locationData?.nodes.map(location => ({
       label: formatLocationLabel(location),
       value: location.id,
     })) || [];
 
+  if (data && data?.locations.nodes.length) {
+    const assignedLocations = data.locations.nodes.map(location => ({
```

Comment:
A comment might be helpful here - assigned ones are filtered out by API, then added here, then filtered by the dropdown - good to explain why we add here?